src/schema/users.ts

```diff
@@ -147,16 +151,21 @@ export const typeDefs = gql`
     """
     Get the reading rank of the user
     """
-    userReadingRank(id: ID!): ReadingRank
+    userReadingRank(id: ID!, timezone: String): ReadingRank
     """
     Get the reading rank history of the user.
     An aggregated count of all the ranks the user ever received.
     """
-    userReadingRankHistory(id: ID!): [ReadingRankHistory]
+    userReadingRankHistory(id: ID!, timezone: String): [ReadingRankHistory]
     """
     Get a heatmap of reads per day in a given time frame.
     """
-    userReadHistory(id: ID!, after: String!, before: String!): [ReadHistory]
+    userReadHistory(
+      id: ID!
+      timezone: String
+      after: String!
+      before: String!
+    ): [ReadHistory]
```

Comment:
I think the timezone here should be queried from the database according to the user id instead of receiving it in the request. This also makes sure the right timezone is used